frontend/src/components/TriggeredPrompt.tsx

```diff
@@ -72,6 +74,10 @@ const TriggeredPrompt: FunctionComponent<TriggeredPromptProps> = ({
   const prompt1 = new Audio(prompt1Url);
   const prompt2 = new Audio(prompt2Url);
 
+  prompt2.onended = () => {
+    secondPromptFinishedHandler?.();
```

Comment:
Do you need the `.` here? Looks kinda weird, but might be needed

Reply:
Agreed it looks a little odd, but it is needed